src/lib/InvoicePrint.svelte

```diff
@@ -18,6 +20,9 @@ onMount(() => {
     to = $invoice.to.split("\n");
     notes = $invoice.notes.split("\n");
 
+    logoSrc = $invoice.logoBase64Img; // needed here because of https://svelte.dev/docs/svelte/runtime-warnings#Client-warnings-hydration_attribute_changed
+    logoAlt = $invoice.logoFilename; // same as above, svelte 5 changes the way image src works on hydration
```

Comment:
they changed the image hydration too, if this was not done this way in svelte 5, invoice logo doesn't show up :angry: